src/components/sections/integration/grid/index.tsx

```diff
@@ -0,0 +1,5 @@
+import { StyledIntegrationGrid } from './grid.styles';
+
+export const IntegrationGrid = ({ children }: Component) => {
+  return <StyledIntegrationGrid>{children}</StyledIntegrationGrid>;
+};
```

Comment:
Do you think we can make this component receive an array of integrations instead? Something like:

```suggestion
type IntegrationGridProps = {
  integrations: IntegrationCardProps[];
};

export const IntegrationGrid = ({ integrations }: Component<IntegrationGridProps>) => {
  return (
    <StyledIntegrationGrid>
      {Array.isArray(integrations) &&
        integrations.length > 0 &&
        integrations.map(integration => <IntegrationCard {...integration} />)}
    </StyledIntegrationGrid>
  );
};
```